sdk/core/Azure.Core.TestFramework/src/Generated/TestProxyRestClient.cs

```diff
@@ -249,6 +249,10 @@ internal HttpMessage CreateStopRecordRequest(string xRecordingId, IDictionary<st
             uri.AppendPath("/record/stop", false);
             request.Uri = uri;
             request.Headers.Add("x-recording-id", xRecordingId);
+            if (xRecordingSkip != null)
+            {
+                request.Headers.Add("x-recording-skip", xRecordingSkip);
```

Comment:
What values would be valid for `xRecordingSkip`? If it's just a `bool`, should we make it a `bool` and `ToString` it here?

Reply:
It could be a bool, but the Test Proxy is expecting "request-response" as the value, and this is generated code.